app/controllers/concerns/make_result_index.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class MakeResultIndex
+  def make_index(cls, collection, query_params)
+    cls.new '/vi/images', ActionController::Parameters.new(query_params), collection
```

Comment:
This needs to be set to the path of your application - `v1` is the version namespace, so `/v1/snippets` or whatever.